go.mod

```diff
@@ -53,7 +53,7 @@ require (
 	code.cloudfoundry.org/bbs v0.0.0-20200403215808-d7bc971db0db
 	code.cloudfoundry.org/garden v0.0.0-20210208153517-580cadd489d2
 	code.cloudfoundry.org/lager v2.0.0+incompatible
-	github.com/DataDog/agent-payload/v5 v5.0.12
+	github.com/DataDog/agent-payload/v5 v5.0.16-0.20220216110510-0289438e2c68
```

Comment:
It needs to be updated once this RP https://github.com/DataDog/agent-payload/pull/158 got merged